servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.cpp

```diff
@@ -1578,6 +1578,11 @@ void RenderForwardClustered::_render_scene(RenderDataRD *p_render_data, const Co
 
 	RENDER_TIMESTAMP("Prepare 3D Scene");
 
+	// get info about our rendering effects
+	bool re_needs_motion_vectors = _compositor_effects_has_flag(p_render_data, RS::COMPOSITOR_EFFECT_FLAG_NEEDS_MOTION_VECTORS);
```

Comment:
I hate to nitpick on naming style. But can we rename these to `effect_**`?

At the very least `re` should change to `ce` now that they are called compositor effects. It took me an embarressinly long time to figure out that `re` stood for "rendering effect".

Reply:
hehehehe, shows you how long ago I wrote this code. I chose against effect because it's so ambiguous (we have many effects, which one are we referring to here), while using `rendering_effect_` as a prefix is rather, uhm, long....

With all of Darios changes there is even an argument whether we should keep the variables, in many cases we now only check them once. there are only one or two that are checked multiple times.

Reply:
Actually found a few more are needed until we automate resolves better. So just went for the `ce_` prefix for now.